client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main.tsx

```diff
@@ -44,52 +45,56 @@ const ScreenMain = ( {
 				hideBack
 			/>
 			<div className="screen-container__body screen-container__body--align-sides">
-				<ItemGroup>
-					<NavigationButtonAsItem
-						path="/header"
-						icon={ header }
-						aria-label={ translate( 'Header' ) }
-						onClick={ () => onSelect( 'header' ) }
-					>
-						<span className="pattern-layout__list-item-text">{ translate( 'Header' ) }</span>
-					</NavigationButtonAsItem>
-					<NavigationButtonAsItem
-						path="/section"
-						icon={ layout }
-						aria-label={ translate( 'Sections' ) }
-						onClick={ () => onSelect( 'section' ) }
-					>
-						<span className="pattern-layout__list-item-text">{ translate( 'Sections' ) }</span>
-					</NavigationButtonAsItem>
-					<NavigationButtonAsItem
-						path="/footer"
-						icon={ footer }
-						aria-label={ translate( 'Footer' ) }
-						onClick={ () => onSelect( 'footer' ) }
-					>
-						<span className="pattern-layout__list-item-text">{ translate( 'Footer' ) }</span>
-					</NavigationButtonAsItem>
-					{ isEnabled( 'pattern-assembler/color-and-fonts' ) && (
-						<>
-							<NavigationButtonAsItem
-								path="/color-palettes"
-								icon={ color }
-								aria-label={ translate( 'Colors' ) }
-								onClick={ () => onSelect( 'color-palettes' ) }
-							>
-								<span className="pattern-layout__list-item-text">{ translate( 'Colors' ) }</span>
-							</NavigationButtonAsItem>
-							<NavigationButtonAsItem
-								path="/font-pairings"
-								icon={ typography }
-								aria-label={ translate( 'Fonts' ) }
-								onClick={ () => onSelect( 'font-pairings' ) }
-							>
-								<span className="pattern-layout__list-item-text">{ translate( 'Fonts' ) }</span>
-							</NavigationButtonAsItem>
-						</>
-					) }
-				</ItemGroup>
+				<HStack direction="column" alignment="top" spacing="4">
+					<NavigatorItemGroup title={ translate( 'Layout' ) }>
+						<NavigationButtonAsItem
+							path="/header"
+							icon={ header }
+							aria-label={ translate( 'Header' ) }
+							onClick={ () => onSelect( 'header' ) }
+						>
+							<span className="pattern-layout__list-item-text">{ translate( 'Header' ) }</span>
+						</NavigationButtonAsItem>
+						<NavigationButtonAsItem
+							path="/section"
+							icon={ layout }
+							aria-label={ translate( 'Sections' ) }
+							onClick={ () => onSelect( 'section' ) }
+						>
+							<span className="pattern-layout__list-item-text">{ translate( 'Sections' ) }</span>
+						</NavigationButtonAsItem>
+						<NavigationButtonAsItem
+							path="/footer"
+							icon={ footer }
+							aria-label={ translate( 'Footer' ) }
+							onClick={ () => onSelect( 'footer' ) }
+						>
+							<span className="pattern-layout__list-item-text">{ translate( 'Footer' ) }</span>
+						</NavigationButtonAsItem>
+					</NavigatorItemGroup>
+					<NavigatorItemGroup title={ translate( 'Style' ) }>
+						{ isEnabled( 'pattern-assembler/color-and-fonts' ) && (
```

Comment:
The feature flag check should be outside of `<NavigatorItemGroup title={ translate( 'Style' ) }>`

Reply:
Thanks for the comment! https://github.com/Automattic/wp-calypso/pull/75321/commits/c87ab39f06dfe72f5c32d35b0ad4bc211533fe85